anomalib/utils/callbacks/visualizer_callback.py

```diff
@@ -68,7 +69,7 @@ def _add_images(
         for log_to in module.hparams.project.log_images_to:
             if log_to in loggers.AVAILABLE_LOGGERS:
                 # check if logger object is same as the requested object
-                if log_to in logger_type and module.logger is not None:
+                if log_to in logger_type and module.logger is not None and isinstance(module.logger, ImageLoggerBase):
```

Comment:
Can you explain why this is needed?

Reply:
it is to silence mypy. Some typings have changed in pl 1.6, causing some mypy issues.

Reply:
Alright, so no functional changes to the visualizer callback then